cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -360,23 +363,27 @@ func (oi *OVFImporter) setUpImportWorkflow() (workflow *daisy.Workflow, err erro
 	return workflow, nil
 }
 
-// Import runs OVF import
+// Import runs OVF import. It first imports all of the disk files to images, and then uses the images
+// to create an instance. If the user requested a machine image, then it converts the instance to a machine image.
 func (oi *OVFImporter) Import() error {
 	oi.Logger.User("Starting OVF import workflow.")
 	if err := oi.paramValidator.ValidateAndPopulate(oi.params); err != nil {
 		return err
 	}
-	if err := oi.setupWorker().Run(map[string]string{}); err != nil {
+	if err := oi.importDiskFilesToImages(); err != nil {
+		return err
+	}
+	if err := oi.createWorkerForFinalInstance().Run(map[string]string{}); err != nil {
 		oi.Logger.User(err.Error())
 		return err
 	}
 	oi.Logger.User("OVF import workflow finished successfully.")
 	return nil
 }
 
-func (oi *OVFImporter) setupWorker() daisyutils.DaisyWorker {
+func (oi *OVFImporter) createWorkerForFinalInstance() daisyutils.DaisyWorker {
```

Comment:
`ResourceLabeler` should label all temporary and final resources. I'm not sure if this feature was potentially lost in the DaisyWorker refactor earlier, but it seems it would be lost after this PR is merged.

Reply:
All of the workflows in CLI tools are now executed through DaisyWorker, which enforces a resource labeler to be present:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/a9f351a2d3e296536bd521b58afe6604f7f6db21/cli_tools/common/utils/daisyutils/daisy_worker.go#L70

I ran an OVF import with this PR, and here are the labels I'm seeing for the inflation step:

**instance**
```
  "labels": {
    "instance-import-build-id": "ovf-tss8d49yhw-1",
    "instance-import-tmp": "true"
  },
```
**boot disk**
```
  "labels": {
    "instance-import-build-id": "ovf-tss8d49yhw-1",
    "instance-import-tmp": "true"
  },
```

**scratch disk**
```
  "labels": {
    "instance-import-build-id": "ovf-tss8d49yhw-1",
    "instance-import-tmp": "true"
  },
```

**translation disk**
```
  "labels": {
    "instance-import-build-id": "ovf-tss8d49yhw-1",
    "instance-import-tmp": "true"
  },
```